### ISSUE TITLE
Provide more guidance for writing long descriptions

### ISSUE THREAD

**Issue by avneeshsingh** [IDPF/epubtestweb#69]
Long descriptions can be a very useful place to collect information that cannot be collected with objective test criteria. for example usability, intuitive UI, easy to use documentation for accessibility features, stability of product etc.

One way to get more structured comments can be to ask specific questions after the testers completes the test form. Each question can follow by an text field where testers can write their comments.
